src/deep-cache/lib/SharedCache.js

```diff
@@ -56,13 +58,12 @@ export class SharedCache {
   /**
    * @param {Request} request
    * @returns {String}
-   * @private
    */
   buildKeyFromRequest(request) {
     let action = request.action;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).](https://www.codacy.com/app/MitocGroup/deep-framework/file/1784383532/issues/source?bid=2895475&fileBranchId=3068904#l63)